extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayBuildConfig.java

```diff
@@ -18,4 +18,10 @@
      */
     @ConfigItem
     public Optional<List<String>> locations;
+
+    /**
+     * Whether or not an healtcheck is published in case the smallrye-health extension is present (default to true).
```

Comment:
```suggestion
     * Whether or not a health check is published in case the smallrye-health extension is present.
```
The default will be automatically included in the generated documentation.